app/scripts/controllers/main/MainController.js

```diff
@@ -2,7 +2,8 @@
     mifosX.controllers = _.extend(module, {
         MainController: function (scope, location, sessionManager, translate, $rootScope, localStorageService, keyboardManager, $idle, tmhDynamicLocale,
                   uiConfigService, $http) {
-            $http.get('release.json').success(function(data) {
+            $http.get('release.json').then(function(data) {
```

Comment:
```suggestion
            $http.get('release.json').then(function(response) {
            var data = response.data;
```

Refer : https://docs.angularjs.org/guide/migration#-http-

Note: The method signatures and return values are different. In Angular v1.5 success method used to return the following data:
`$http(...).
success(function onSuccess(data, status, headers, config) {
  // Handle success
  ...
}).
error(function onError(data, status, headers, config) {
  // Handle error
  ...
});`

but now it returns a single object containing the above mentioned property :

```
$http(...).
  then(function onSuccess(response) {
    // Handle success
    var data = response.data;
    var status = response.status;
    var statusText = response.statusText;
    var headers = response.headers;
    var config = response.config;
    ...
  }).
  catch(function onError(response) {
    // Handle error
    var data = response.data;
    var status = response.status;
    var statusText = response.statusText;
    var headers = response.headers;
    var config = response.config;
    ...
  });
```